File: conciertosBack/shared/src/main/java/com/rubenazo/buscaConciertos/domain/SalaConcierto.java
package com.rubenazo.buscaConciertos.domain;

import java.time.Instant;

public record SalaConcierto(
    String id,
    String name,
    String address,
    String city,
    String province,
    Double lat,
    Double lng,
    String imageUrl,
    String description,
    String sourceUrl,
    Instant updatedAt
) {}

File: conciertosFront/shared/src/commonMain/kotlin/org/rubenazo/conciertosfront/core/domain/model/SalaConcierto.kt
package org.rubenazo.conciertosfront.core.domain.model

data class SalaConcierto(
    val id: String,
    val name: String,
    val address: String?,
    val city: String,
    val province: String,
    val lat: Double?,
    val lng: Double?,
    val imageUrl: String?,
    val description: String?,
    val sourceUrl: String?
)
